cumulusci/robotframework/Salesforce.py

```diff
@@ -1100,27 +1105,15 @@ def wait_until_salesforce_is_ready(self, locator=None, timeout=None, interval=5)
                     self.selenium.capture_page_screenshot()
                     raise Exception("Timed out waiting for a lightning page")
```

Comment:
Will this result in two screenshots due to the added use of the `capture_screenshot_on_error` decorator?

Reply:
Yes it will. Good catch.